MidCamp/midcamp
# Allow Sponsors to Create Jobs

## Testing 
* Navigate to `/node/add/sponsor`
* Fill in the fields
* Save & Publish
* Create a user with the role "Sponsor"
* Masquerade as said user
* Navigate to `/node/add/job`
* Fill in the fields
* Click Save & Publish
* Navigate to `/jobs`
* Confirm you see the teaser for new job
